01-node-tutorial/answers/10-fs-sync.js

```diff
@@ -0,0 +1,25 @@
+const { readFileSync, writeFileSync } = require("fs");
+// const fs = require('fs');
+// fs.readFileSync()
+
+const first = readFileSync("./temporary/content/first.txt", "utf8");
+const second = readFileSync("./temporary/content/second.txt", "utf8");
```

Comment:
Hello, great job here, 
however, you need to use readFileSync to read the contents of the file you wrote into after the writeFileSync line..
so you can put the readFileSync() line below the writeFileSync and ensure it's the exact same file path